.zuul/playbooks/run.yaml

```diff
@@ -0,0 +1,42 @@
+- hosts: all
+  become: yes
+  roles:
+  - config-golang
+  - export-cloud-openrc
```

Comment:
- role: export-cloud-openrc
      vars:
        cloud_name: 'citynetwork'

Reply:
Done

Reply:
Looks like there is still something wrong:

```
hosts": {
              "ubuntu-xenial-citynetwork": {
                "action": "set_fact",
                "censored": "the output has been hidden due to the fact that 'no_log: true' was specified for this result",
                "failed": true
              }
            },
            "role": {
              "id": "fa163ef5-08f0-6cb1-7936-000000000012",
              "name": "export-cloud-openrc",
              "path": "/var/lib/zuul/builds/c475a77f846c4066ab87b0af8306f1c4/untrusted/project_1/github.com/theopenlab/openlab-zuul-jobs/roles/export-cloud-openrc"
            },
            "task": {
              "duration": {
                "end": "2020-01-13T18:36:12.137608Z",
                "start": "2020-01-13T18:36:12.069078Z"
              },
              "id": "fa163ef5-08f0-6cb1-7936-00000000001a",
              "name": "Set fact for citynetwork openrc"
            }
```